### DISCUSSION TITLE
scenario profiel option

### DISCUSSION THREAD

**Discussion by PoepieFloepie** [seerge/g-helper#1981] [Ideas]
is it not possible add  a scenario profiel with apps that launch for example turbo mode when you start the app
like in armoury crate

<img width="1104" alt="Schermafbeelding 2024-01-25 182409" src="https://github.com/seerge/g-helper/assets/46219717/152bef06-72ac-4078-ad22-af081f385055">

**Comment by seerge** [comment]
@PoepieFloepie hello, you can't, but it's extremely easy to switch modes in the app forwards and backwards. 

Also there are custom hotkeys for the first 5 profiles (including custom) https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings

And if you are using some kind of a game launcher or so, you can assign it to trigger that hotkey on launch :)
